### DISCUSSION TITLE
Add "use current" to grid settings in editor

### DISCUSSION THREAD

**Discussion by Alib234** [ppy/osu#29818] [Ideas] [answered]
it is quite annoying to manually have to change the grid position to an object's, while this use case is probably not common and possibly won't ever be common, it would make it better for people who want to change the grid position to a specific object's position.
I could see this being annoying to implement on sliders as a user could want the grid to be set at the slider head or tail and those aren't separately selectable.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/30069
